### PR TITLE
open sarama error and update sarama read timeout

### PR DESCRIPTION
**Pull request by 3pointer**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: tidb-binlog/driver/reader/reader.go [hunk 1]
```diff
@@ -118,7 +125,11 @@ func (r *Reader) Messages() (msgs <-chan *Message) {
 }
 
 func (r *Reader) getOffsetByTS(ts int64) (offset int64, err error) {
-	seeker, err := NewKafkaSeeker(r.cfg.KafkaAddr, nil)
+	conf := sarama.NewConfig()
+	// set to 5 minutes to prevent i/o timeout in read huge message
```
Comment (suzaku) [inline tidb-binlog/driver/reader/reader.go]:
```suggestion
	// set to 5 minutes to prevent i/o timeout when reading huge message
```

**Review by suzaku — COMMENTED** [review]
rest LGTM

**Review by suzaku — COMMENTED** [review]
LGTM

**Review by july2993 — DISMISSED** [review]

**Review by july2993 — DISMISSED** [review]

**Review by july2993 — DISMISSED** [review]

File: tidb-binlog/driver/reader/offset.go [hunk 2]
```diff
@@ -198,7 +199,15 @@ func (ks *KafkaSeeker) getTSAtOffset(topic string, partition int32, offset int64
 		case msg := <-pc.Errors():
 			err = msg.Err
 			time.Sleep(time.Second)
-			continue
```
Comment (IANTHEREAL) [inline tidb-binlog/driver/reader/offset.go]: output log before L201

**Comment by IANTHEREAL** [comment]
Rest LGTM

**Review by IANTHEREAL — APPROVED** [review]
LGTM

**Pull request merged**